snaps/reference/resources.md

```diff
@@ -0,0 +1,74 @@
+---
+description: See more Snaps resources.
+---
+
+# Resources
+
+View the following Snaps resources in addition to this documentation site.
+
+## Primary resources
+
+- [Snaps homepage](https://metamask.io/snaps/)
+- [Snaps directory](https://snaps.metamask.io/) - A directory of allowlisted Snaps you can try.
+- [MetaMask Flask](https://metamask.io/flask/) - Install Flask to build your own Snaps and run Snaps locally.
+- [SIPs](https://github.com/MetaMask/SIPs) - Snaps Improvement Proposals.
+- [Open Beta Readiness Guide](https://github.com/MetaMask/snaps/discussions/1411) - A guide to get
+  your Snap allowlisted on MetaMask.
+
+## Example Snaps
+
+- [Example Snaps](https://github.com/MetaMask/snaps/tree/main/packages/examples) - A directory of
+  Snaps maintained by MetaMask.
+- [Dogecoin Snap](https://github.com/ziad-saab/dogecoin-snap) - A tutorial for creating a Snap to
+  add Dogecoin to MetaMask.
+- [Starknet Snap](https://github.com/Consensys/starknet-snap) - A Snap to add Starknet to MetaMask.
+- [MobyMask Phishing Warning Snap](https://github.com/Montoya/mobymask-snap) - A Snap that warns you
+  when interacting with phishing contracts.
+- [Mystery Snap](https://github.com/Montoya/random-snap) - A Snap that returns random numbers and answers.
+- [Snappy Recovery](https://github.com/ziad-saab/snappy-recovery) - A Snap that enables social
+  recovery of private keys.
+- [Pet Fox game](https://github.com/Montoya/pet-fox) - A simple game built using a Snap.
+
+## Developer tools
+
+- [Template Snap](https://github.com/MetaMask/template-snap-monorepo) - A rich template that
+  includes TypeScript/React and vanilla JavaScript options, a CLI for building, packaging, and
+  deploying your Snap, and a companion dapp UI you can build on.
+- [Snaps Simulator](https://metamask.github.io/snaps/snaps-simulator/latest) - A tool for simulating
+  Snaps in the browser, streamlining the development process.
+- [Test Snaps](https://github.com/MetaMask/test-snaps) - A collection of test Snaps and a dapp for
```

Comment:
This was moved into the Snaps monorepo, it's at https://github.com/MetaMask/snaps/tree/main/packages/test-snaps and the dapp is at https://metamask.github.io/snaps/test-snaps/latest